src/components/tests/content-tab-overview-shortcuts-section.test.tsx

```diff
@@ -16,11 +23,53 @@ const selectedSite: StartedSiteDetails = {
 };
 
 const mockGetIpcApi = getIpcApi as jest.Mock;
-jest.mock( 'src/hooks/use-check-installed-apps' );
 jest.mock( 'src/lib/get-ipc-api' );
 jest.mock( 'src/hooks/use-theme-details' );
 jest.mock( 'src/hooks/use-feature-flags' );
 
+// Create a test reducer for installedAppsApi
+function installedAppsTestReducer( state: RootState, action: UnknownAction ) {
+	if ( action.type === 'installedApps/setInstalledApps' ) {
+		const payload = action.payload as {
+			installedApps: InstalledAppsState;
+		};
+
+		return produce( state!, ( draftState ) => {
+			if ( draftState ) {
+				// Set the query result in the RTK Query cache
+				draftState.installedAppsApi.queries = {
+					'getInstalledApps({"forceRefetch":false})': {
+						status: QueryStatus.fulfilled,
+						data: payload.installedApps,
+						error: undefined,
+						requestId: 'test-request-id',
+						endpointName: 'getInstalledApps',
+						startedTimeStamp: 0,
+						fulfilledTimeStamp: 0,
+						originalArgs: undefined as never,
+					},
+				};
+			}
+		} );
+	}
+
+	return testReducer( state, action );
+}
+
+// Create test actions for installedAppsApi
+const installedAppsTestActions = {
+	setInstalledApps: ( installedApps: InstalledAppsState ) => {
+		return { type: 'installedApps/setInstalledApps', payload: { installedApps } };
+	},
+};
+
+// Replace the store's reducer with our test reducer
+store.replaceReducer( installedAppsTestReducer as Reducer< RootState > );
```

Comment:
```suggestion
store.replaceReducer( testReducer );
```
Looks like `installedAppsTestReducer` isn't used here, and we can stick with just `testReducer`

Reply:
Nice catch, thanks!